packages/common/src/utils/timeFrames.ts

```diff
@@ -242,103 +242,123 @@ type TimeFrameConfig = {
         originalSql: string,
         type: DimensionType,
     ) => string;
+    getAxisMinInterval: () => number | null;
 };
 
 export const timeFrameConfigs: Record<TimeFrames, TimeFrameConfig> = {
     RAW: {
         getLabel: () => 'Raw',
         getDimensionType: (fallback) => fallback,
         getSql: (_adapterType, _timeInterval, originalSql) => originalSql,
+        getAxisMinInterval: () => null,
     },
     MILLISECOND: {
         getLabel: () => 'Millisecond',
         getDimensionType: () => DimensionType.TIMESTAMP,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     SECOND: {
         getLabel: () => 'Second',
         getDimensionType: () => DimensionType.TIMESTAMP,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     MINUTE: {
         getLabel: () => 'Minute',
         getDimensionType: () => DimensionType.TIMESTAMP,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     HOUR: {
         getLabel: () => 'Hour',
         getDimensionType: () => DimensionType.TIMESTAMP,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     DAY: {
         getLabel: () => 'Day',
         getDimensionType: () => DimensionType.DATE,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     WEEK: {
         getLabel: () => 'Week',
         getDimensionType: () => DimensionType.DATE,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => null,
     },
     MONTH: {
         getLabel: () => 'Month',
         getDimensionType: () => DimensionType.DATE,
         getSql: getSqlForTruncatedDate,
+        getAxisMinInterval: () => 2629800000,
```

Comment:
what is this magic number ??? :exploding_head:

Reply:
how many milliseconds there are in a month